mountpoint-s3-fs/examples/prefetch_benchmark.rs

```diff
@@ -233,9 +233,9 @@ fn main() -> anyhow::Result<()> {
         let received_size = received_bytes.load(Ordering::SeqCst);
         total_bytes += received_size;
         println!(
-            "{iteration}: received {received_size} bytes in {:.2}s: {:.2} Gib/s",
+            "{iteration}: received {received_size} bytes in {:.2}s: {:.2} Gb/s",
             elapsed.as_secs_f64(),
-            (received_size as f64) / elapsed.as_secs_f64() / (1024 * 1024 * 1024 / 8) as f64
+            (received_size as f64) / elapsed.as_secs_f64() / (1000 * 1000 * 1000 / 8) as f64
```

Comment:
nit: can we make this value more readable? not sure if best to use a clearly named variable or a function to reuse in line 251 too